### PR TITLE
Create MemoryPipelineChannelCreatorTest.java

### PR DESCRIPTION
**Pull request by sunkai-cai**
complete #15702.

Changes proposed in this pull request:
- add MemoryPipelineChannelCreator test （MemoryPipelineChannelFactory rename from MemoryPipelineChannelFactory）
- PositionInitializerFactory and PipelineSQLBuilderFactory has test

### CODE REVIEW

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/test/java/org/apache/shardingsphere/data/pipeline/core/ingest/channel/memory/MemoryPipelineChannelCreatorTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import java.util.Properties;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public final class MemoryPipelineChannelCreatorTest {
+
+    @Test
+    public void testInit() throws Exception {
```
Comment (terrymanu) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/test/java/org/apache/shardingsphere/data/pipeline/core/ingest/channel/memory/MemoryPipelineChannelCreatorTest.java:33]: Please use `assert` as prefix for test method

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/test/java/org/apache/shardingsphere/data/pipeline/core/ingest/channel/memory/MemoryPipelineChannelCreatorTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        result.setProperty("block-queue-size", "200");
+        MemoryPipelineChannelCreator memoryPipelineChannelCreator = new MemoryPipelineChannelCreator();
+        memoryPipelineChannelCreator.init(result);
+        assertThat(memoryPipelineChannelCreator.getProps(), is(result));
+        Integer blockQueueSize = ReflectionUtil.getFieldValue(memoryPipelineChannelCreator, "blockQueueSize", Integer.class);
+        assertThat(blockQueueSize, is(200));
+    }
+
+    @Test
+    public void testCreatePipelineChannel() {
```
Comment (terrymanu) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/test/java/org/apache/shardingsphere/data/pipeline/core/ingest/channel/memory/MemoryPipelineChannelCreatorTest.java:44]: Please use `assert` as prefix for test method

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/test/java/org/apache/shardingsphere/data/pipeline/core/ingest/channel/memory/MemoryPipelineChannelCreatorTest.java (hunk 2 above)
Comment (sunkai-cai) [reply shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/test/java/org/apache/shardingsphere/data/pipeline/core/ingest/channel/memory/MemoryPipelineChannelCreatorTest.java:44]: ok

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/test/java/org/apache/shardingsphere/data/pipeline/core/ingest/channel/memory/MemoryPipelineChannelCreatorTest.java (hunk 1 above)
Comment (sunkai-cai) [reply shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/test/java/org/apache/shardingsphere/data/pipeline/core/ingest/channel/memory/MemoryPipelineChannelCreatorTest.java:33]: ok

**Review by terrymanu — APPROVED** [review]

**Pull request merged**